package.json

```diff
@@ -48,6 +48,8 @@
     "extend": "^3.0.2",
     "google-gax": "^4.0.5",
     "is": "^3.3.0",
+    "proto3-json-serializer": "^2.0.1",
```

Comment:
These are pretty big additions. We use protobufjs & proto3-json-serializer in google-gax, is it possible to just use it from there?

Reply:
Yes. Great suggestion.